iron-list.html

```diff
@@ -1732,7 +1726,7 @@
 
     _removeFocusedItem: function() {
       if (this._offscreenFocusedItem) {
-        Polymer.dom(this).removeChild(this._offscreenFocusedItem);
+        Polymer.dom(this._userTemplate.parentNode).removeChild(this._offscreenFocusedItem);
```

Comment:
I assume by `node` you actually mean `this._offscreenFocusedItem` here since `node` isn't defined.